Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG OSDB_BKC_VERSION=
 ARG HASVER=${ROCMVERSION:+$ROCMVERSION}
 ARG HASVER=${HASVER:-$OSDB_BKC_VERSION}
 
-ARG BASEIMAGE=rocm/miopen:ci_c1ca2a
+ARG BASEIMAGE=rocm/miopen:ci_5fec03
```

Comment:
Can this be passed as an environmental variable + docker run arg instead?

Reply:
BASEIMAGE is passable as a docker-arg